scripts/ci-conformance.sh

```diff
@@ -92,12 +92,35 @@ AZURE_SSH_PUBLIC_KEY=$(< "${AZURE_SSH_PUBLIC_KEY_FILE}" tr -d '\r\n')
 export AZURE_SSH_PUBLIC_KEY
 
 cleanup() {
+    # clean up GMSA NODE RG
+    if [[ "${SKIP_CLEANUP:-}" != "true" && -n ${GMSA_ID:-} ]]; then
+      echo "Cleaning up gMSA resources $GMSA_NODE_RG with keyvault $CI_RG-gmsa"
+      az keyvault secret list --vault-name "$CI_RG"-gmsa --query "[? contains(name, '${GMSA_ID}')].name" -o tsv | while read -r secret ; do
+         az keyvault secret delete -n "$secret" --vault-name "$CI_RG"-gmsa
+      done
+
+      az group delete --name "$GMSA_NODE_RG" --no-wait -y --force-deletion-types=Microsoft.Compute/virtualMachines,Microsoft.Compute/virtualMachineScaleSets
+    fi
+
     "${REPO_ROOT}/hack/log/redact.sh" || true
 }
 
 trap cleanup EXIT
 
 if [[ "${WINDOWS}" == "true" ]]; then
+  if [[ $KUBETEST_WINDOWS_CONFIG =~ "windows-serial-slow" ]]; then
```

Comment:
Is this bit documented anywhere?
I can image someone having a bad day trying to troubleshoot why GMSA tests aren't running somewhere in PROW only to find this conditional...
Maybe we could at least add a log line like 'Skipping GMSA configuration' if we aren't performing the config to help debugging?

Reply:
It isn't. We currently run the GMSA tests in the serial/slow jobs.  There isn't a strict requirement for this and in fact the tests are pretty fast, just to initial set up of the cluster and domain is slow.  

I used this because i didn't want to introduce yet another ENV but maybe it would be better to have it as additional setup? It would make your suggestion and debugging simpler.